lib/inv_cg_quda.cpp

```diff
@@ -427,6 +427,7 @@ namespace quda {
     // set this to true if maxResIncrease has been exceeded but when we use heavy quark residual we still want to continue the CG
     // only used if we use the heavy_quark_res
     bool L2breakdown = false;
+    const double L2breakdown_eps = 100. * uhigh;
```

Comment:
Where does the factor of 100 come from?

Reply:
Somewhat handwaving. We will never reach machine eps and while we might be able to get with in 10 * uhigh it seemed to tight for the purpose here. So just tried 100 ...